tests/route/test_route_consistency.py

```diff
@@ -248,12 +248,8 @@ def test_bgpd_crash_and_recover(self, duthosts):
                     assert num_routes_withdrawn == len(self.pre_test_route_snapshot[dut_instance_name] -
                                                        post_withdraw_route_snapshot[dut_instance_name])
 
-            logger.info("start bgpd for {}".format(duthost.hostname))
-            for id in namespace_ids:
-                if id is None:
-                    id = ""
-                duthost.shell("docker exec {} supervisorctl start {}".format("bgp" + str(id), "bgpd"))
-                duthost.shell("docker exec {} supervisorctl restart {}".format("bgp" + str(id), "bgpcfgd"))
+            logger.info("recover bgpd for {}".format(duthost.hostname))
+            config_reload(duthost)
```

Comment:
why change the bgp restart to config reload?

Reply:
This is to make it consistent with the recovery method used in the other test cases. Also, this can ensure the complete recovery of the whole containers group.